InvenTree/InvenTree/api_version.py

```diff
@@ -2,18 +2,11 @@
 
 
 # InvenTree API version
-INVENTREE_API_VERSION = 141
+INVENTREE_API_VERSION = 139
```

Comment:
@matmair looks like this PR is out of sync and will revert some other recent changes?

Reply:
All the different active branches are beginning to become a problem, cleaning up rn.